src/client/html/style.css

```diff
@@ -0,0 +1,777 @@
+  text-align: center;
+}
+
+#team1 button {
+  color: var(--red-bright);
+}
+
+#team2 button {
+  color: var(--blue-bright);
+}
+
+.team button:hover {
+  background-color: var(--gold);
+  color: var(--black) !important;
+}
+
+#board .separator {
+  content: "";
+  width: 100%;
+}
+
+#board button {
+  display: flex;
+  flex: 1 0 19%;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  padding: 16px;
+
+  width: 100px;
+  height: 64px;
+  border: 2px solid var(--black);
+  box-shadow: 0px 4px 0px var(--black);
+  border-radius: 16px;
+  background: linear-gradient(
+      106.36deg,
+      #f7eede 12.54%,
+      #ffffff 12.99%,
+      #ffffff 17.86%,
+      #f7eede 18.05%,
+      #f7eede 23.55%,
+      #ffffff 23.59%,
+      #ffffff 31.3%,
+      #f7eede 31.38%,
+      #f7eede 82.94%,
+      #ffffff 82.98%,
+      #ffffff 87.8%,
+      #f7eede 88.15%
+    ),
+    #f7eede;
+  background-size: 164px 64px, 164px 64px;
+  font-family: var(--font-family);
+  font-style: normal;
+  font-weight: 900;
+  font-size: 16px;
+  line-height: 27px;
+  text-align: center;
+  text-transform: uppercase;
+  color: var(--black);
+  opacity: 1;
+}
+
+#board button.revealed {
+  box-shadow: 0px 4px 0px var(--black) inset;
+  opacity: 0.8;
+  color: var(--white);
+}
+
+#board button.neutral.revealed {
+  color: var(--black);
+}
+
+#board button.team2 {
+  color: var(--white);
+  background: linear-gradient(
+      106.36deg,
+      #3988ff 12.54%,
+      #95b9ef 12.99%,
+      #95b9ef 17.84%,
+      #3988ff 17.85%,
+      #3988ff 23.55%,
+      #95b9ef 23.56%,
+      #95b9ef 31.3%,
+      #3988ff 31.31%,
+      #3988ff 82.94%,
+      #95b9ef 82.95%,
+      #95b9ef 87.8%,
+      #3988ff 87.81%
+    ),
+    #3988ff;
+}
+
+#board button.team1 {
+  color: var(--white);
+  background: linear-gradient(
+    106.36deg,
+    #ff0000 12.54%,
+    #ef8686 12.55%,
+    #ef8686 17.86%,
+    #ff0000 17.87%,
+    #ff0000 23.55%,
+    #ef8686 23.56%,
+    #ef8686 31.3%,
+    #ff0000 31.31%,
+    #ff0000 82.92%,
+    #ef8686 82.93%,
+    #ef8686 87.8%,
+    #ff0000 87.81%
+  );
+}
+
+#board button.assassin {
+  background: linear-gradient(
+    106.36deg,
+    #838383 12.54%,
+    #ababab 12.55%,
+    #ababab 17.86%,
+    #838383 17.87%,
+    #838383 23.55%,
+    #ababab 23.56%,
+    #ababab 31.3%,
+    #838383 31.31%,
+    #838383 82.94%,
+    #ababab 82.95%,
+    #ababab 87.8%,
+    #838383 87.81%
+  );
+}
+/*
+#board button.neutral {
+  background: linear-gradient(
+    106.36deg,
+    #b8b8b8 12.54%,
+    #e5e5e5 12.55%,
+    #e5e5e5 17.86%,
+    #b8b8b8 17.87%,
+    #b8b8b8 23.55%,
+    #e5e5e5 23.56%,
+    #e5e5e5 31.3%,
+    #b8b8b8 31.31%,
+    #b8b8b8 82.94%,
+    #e5e5e5 82.95%,
+    #e5e5e5 87.8%,
+    #b8b8b8 87.81%
+  );
+} */
+
+#board button:hover:enabled {
+  opacity: 1;
+  cursor: pointer;
+  border: 2px solid var(--gold);
+  box-shadow: 0px 4px 0px var(--gold);
+  animation: shake 0.82s cubic-bezier(0.36, 0.07, 0.19, 0.97) both infinite;
```

Comment:
I like this shake feature but it makes me feel a little anxious to be honest lol! Can we tone it down just a bit (the speed and how much it moves from side to side?)

Reply:
much better! still makes me heart go pitter patter but that's probably just me 😂